src/style.css

```diff
@@ -1,3 +1,70 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  padding: 0 50px;
+  max-width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  text-align: center;
+  list-style: none;
+  text-decoration: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: flex;
+  color: black;
+  text-decoration: none;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  line-height: 60px;
+  height: 60px;
+  margin-left: 20px;
+}
+
+.is-active {
+  position: relative;
+  margin-left: 0;
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.apple {
```

Comment:
Avoid styling by tag names; use classes instead. This is to ensure that if you have multiple `apple` elements, they don't all get styled the same way unless they have the same class.